Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Prepare the boot build image and dependencies
+FROM clojure:boot-alpine AS boot
+RUN apk add --update git npm
+RUN npm install -g yarn shadow-cljs
+
+# Copy source files into the docker container
+WORKDIR /app
+COPY . ./
+
+# Update game data to the latest release
+RUN git submodule update --init
+RUN cd resources/game \
+ && git checkout master \
+ && git checkout $(git describe --tags --abbrev=0 $(git rev-list --tags --max-count=2) | grep -vE 'continuous')
+
+# Build
+RUN boot build
+
+# Copy build results to the final image
+FROM ruby:alpine
```

Comment:
There is no need to use ruby to serve static files - nginx would handle that perfectly.

Something like this should do the trick:

```Dockerfile
FROM nginx:latest
COPY --from=boot /app/build /usr/share/nginx/html
```

Keep in mind that it will expose port 80, so running it would look like `docker run -p 8000:80`

Reply:
Sure, I just took the suggestion you gave in the README. Updated to nginx, and tested it works.